### DISCUSSION TITLE
my old account no longer has the eye icon

### DISCUSSION THREAD

**Discussion by easygao** [civitai/civitai#400] [General]
![LK1GYZ$BUL7L28DYQZW6H O](https://user-images.githubusercontent.com/20897188/228228872-c6ed82dc-40bc-4c3e-936c-445812206559.png)
When I try to access the Mature Content option, the interface automatically closes and I am unable to turn it on. Please fix this issue so that after I make the selection, the interface either refreshes or keeps the selected option without automatically closing.
I tried using a new account and it was able to display the Mature Content option. The new account has an eye icon in the upper right corner, but my old account no longer has the eye icon.

![$0%HZFX6I6LO3RSU4$90PCR](https://user-images.githubusercontent.com/20897188/228231041-1c483a1e-374c-4a2c-887f-40b6f3004860.png)

**Comment by JustMaier** [comment]
Hey does this seem to be working better for you now?

**Reply by easygao** [reply]
A few days ago, I managed to fix the issue by clearing all browsing history and data, and was able to save my preferences. However, I am not sure if it has been fixed for others.
